### PR TITLE
Replace ws with isomorphic-ws.

### PR DESCRIPTION
**Pull request by jiyoung-an**
Related issues:
- https://github.com/ainblockchain/ain-js/issues/171

### CODE REVIEW

**Comment by jiyoung-an** [comment]
@platfowner when I tried replacing with isomorphic-ws for a local test on browser,
I encountered an error where the websocket connection failed to establish..
I don't know about `35.223.56.37:6000 ` -> this url is get from `net_getEventHandlerNetworkInfo`.

스크린샷 2024-04-11 오전 1 52 50

ps. I'm not sure if this PR will solve the [issue](https://github.com/ainblockchain/ain-js/issues/171), but I'm posting it for reference.

File: src/event-manager/event-channel-client.ts [hunk 1]
```diff
@@ -85,40 +85,44 @@ export default class EventChannelClient {
...
-      });
-      this._wsClient.on('ping', () => {
-        if (this._heartbeatTimeout) {
-          clearTimeout(this._heartbeatTimeout);
-        }
-        this.startHeartbeatTimer(connectionOption.heartbeatIntervalMs || DEFAULT_HEARTBEAT_INTERVAL_MS);
-      });
-      this._wsClient.on('close', () => {
+      };
+      // NOTE(jiyoung): ping is not supported in browser api
```
Comment (platfowner) [inline src/event-manager/event-channel-client.ts:104]: Isn't the goal to support both backend side and browser side? @jiyoung-an

Comment (platfowner) [reply src/event-manager/event-channel-client.ts:104]: maybe browser-or-node package can be used?
https://stackoverflow.com/questions/34550890/how-to-detect-if-script-is-running-in-browser-or-in-node-js

Comment (jiyoung-an) [reply src/event-manager/event-channel-client.ts:104]: In isomorphic-ws, there is no separate ping method, so it seems that a separate message must be sent.
ref) https://github.com/fugle-dev/fugle-marketdata-node/blob/7501aa634f625ec19a101d30c7072cd523439649/src/websocket/client.ts#L45

Comment (jiyoung-an) [reply src/event-manager/event-channel-client.ts]: @platfowner I found reference code with handshake timeout and ping
ref) https://github.com/shapeshift/web/blob/4fd287342223d984f4d499ad171a62297136ec8d/packages/unchained-client/src/websocket.ts#L2

Comment (platfowner) [reply src/event-manager/event-channel-client.ts]: great, thanks! I'll take a look very soon.

**Comment by platfowner** [comment]
I'll work on the custom handshake timeout and ping-pong very soon. They were just added as TODOs in this PR.
Before that, I just tried to upgrade packages and update tests. PTAL

**Review by platfowner — APPROVED** [review]
LGTM with some TODOs.
Once this PR is merged, I'll create a separate PR for the custom handshake timeout and ping-pong. Thanks!

**Comment by jiyoung-an** [comment]
LGTM, thank you for fix broken tests!

**Comment by jiyoung-an** [comment]
Thank you for review! @platfowner now merging..

**Pull request merged**